tools/gen_openvino_dockerfile.py

```diff
@@ -108,12 +108,23 @@ def dockerfile_for_linux(output_file):
 WORKDIR /opt/openvino
 RUN cp -r /workspace/openvino/licensing LICENSE.openvino
 RUN mkdir -p include && \
-    cp -r /workspace/install/runtime/include/ie/* include/. && \
     cp -r /workspace/install/runtime/include/ngraph include/. && \
     cp -r /workspace/install/runtime/include/openvino include/.
 RUN mkdir -p lib && \
-    cp /workspace/install/runtime/lib/intel64/*.so lib/. && \
-    cp /workspace/install/runtime/3rdparty/omp/lib/libiomp5.so lib/.
```

Comment:
Bit of an idle question, is the dependency of iomp completely removed in 2023.0.0?

Reply:
It is moved into the `/.../runtime/lib/intel64` directory:
```
root@...:/workspace/install/runtime/lib/intel64# ls -l
total 3816360
-rw-r--r-- 1 root root    8872422 Jul  6 23:20 cache.json
lrwxrwxrwx 1 root root         20 Jul  6 23:44 libgna.so.3 -> libgna.so.3.5.0.1906
-rw-r--r-- 1 root root    2304416 Mar  6 11:01 libgna.so.3.5.0.1906
-rw-r--r-- 1 root root    2106132 Sep 28  2018 libiomp5.so                           <---------- here
lrwxrwxrwx 1 root root         19 Jul  6 23:44 libopenvino.so -> libopenvino.so.2300
-rw-r--r-- 1 root root  632499720 Jul  6 23:40 libopenvino.so.2023.0.0
lrwxrwxrwx 1 root root         23 Jul  6 23:44 libopenvino.so.2300 -> libopenvino.so.2023.0.0
```
I suspect it is not required for API 2.0, because all the tests are passing without it. Do you think there are any concerns not including it?

Reply:
No concerns from me.